File: exhibition/fix_education.py
import sqlite3

db_path = 'db/six-degrees.sqlite'
conn = sqlite3.connect(db_path)
cursor = conn.cursor()

names = [
    "Omer Barak",
    "Mai Mishli",
    "Shani Libner",
    "Shaked Klett",
    "Shaked Hogi",
    "Yoel Zajdner",
    "Shachar Liz Ben Or",
    "Ravid Dar",
    "Noya Ma'or",
    "Hila Lustig",
    "Sharon Greenberg",
    "Almog Aspaler",
    "Hila Ben Shabbat",
    "Oral Shafsha",
    "Re'ut Birkenstein"
]

for name in names:
    cursor.execute("SELECT id, description, tags FROM people WHERE name = ?", (name,))
    row = cursor.fetchone()
    if not row:
        print(f"Not found: {name}")
        continue
    
    p_id, desc, tags = row
    tags = tags or ''
    desc = desc or ''

    # Clean tags
    tags_list = [t.strip() for t in tags.split(',') if t.strip()]
    filtered_tags = []
    for t in tags_list:
        tl = t.lower()
        if not (tl.startswith('university:') or tl.startswith('degree:') or tl.startswith('graduation year:')):
            filtered_tags.append(t)
    
    filtered_tags.extend([
        'university:University of Haifa',
        'degree:Visual Communication',
        'Graduation Year:2026'
    ])
    new_tags = ', '.join(filtered_tags)

    # Clean desc
    desc_lines = [l.strip() for l in desc.split('\n') if l.strip()]
    filtered_desc = []
    for l in desc_lines:
        ll = l.lower()
        if not (ll.startswith('university:') or ll.startswith('degree:') or ll.startswith('graduation year:')):
            filtered_desc.append(l)
            
    filtered_desc.extend([
        'university: University of Haifa',
        'degree: Visual Communication',
        'Graduation Year: 2026'
    ])
    new_desc = '\n'.join(filtered_desc)

    cursor.execute("UPDATE people SET tags = ?, description = ? WHERE id = ?", (new_tags, new_desc, p_id))
    print(f"Updated: {name}")

conn.commit()
conn.close()
print("Done.")

File: exhibition/fix_education.js
const Database = require('better-sqlite3');
const db = new Database('db/six-degrees.sqlite');

const names = [
    "Omer Barak",
    "Mai Mishli",
    "Shani Libner",
    "Shaked Klett",
    "Shaked Hogi",
    "Yoel Zajdner",
    "Shachar Liz Ben Or",
    "Ravid Dar",
    "Noya Ma'or",
    "Hila Lustig",
    "Sharon Greenberg",
    "Almog Aspaler",
    "Hila Ben Shabbat",
    "Oral Shafsha",
    "Re'ut Birkenstein"
];

for (const name of names) {
    const person = db.prepare('SELECT id, description, tags FROM people WHERE name = ?').get(name);
    if (!person) {
        console.log(`Not found: ${name}`);
        continue;
    }

    let tags = person.tags || '';
    let desc = person.description || '';

    // Remove existing university, degree, graduation year from tags
    let tagsList = tags.split(',').map(t => t.trim()).filter(Boolean);
    tagsList = tagsList.filter(t => {
        const lower = t.toLowerCase();
        return !lower.startsWith('university:') && 
               !lower.startsWith('degree:') && 
               !lower.startsWith('graduation year:');
    });

    // Add new ones
    tagsList.push('university:University of Haifa');
    tagsList.push('degree:Visual Communication');
    tagsList.push('Graduation Year:2026');

    tags = tagsList.join(', ');

    // Remove existing from desc
    let descLines = desc.split('\n').map(l => l.trim());
    descLines = descLines.filter(l => {
        const lower = l.toLowerCase();
        return !lower.startsWith('university:') && 
               !lower.startsWith('degree:') && 
               !lower.startsWith('graduation year:');
    });

    // Add new ones
    descLines.push('university: University of Haifa');
    descLines.push('degree: Visual Communication');
    descLines.push('Graduation Year: 2026');

    desc = descLines.join('\n');

    db.prepare('UPDATE people SET tags = ?, description = ? WHERE id = ?').run(tags, desc, person.id);
    console.log(`Updated: ${name}`);
}
console.log('Done.');
